packages/types/src/clerk.ts

```diff
@@ -737,6 +760,34 @@ export interface Clerk {
    * initiated outside of the Clerk class.
    */
   __internal_setActiveInProgress: boolean;
+
+  /**
+   * This API is in early access and may change in future releases.
+   *
+   * Retrieves all API keys for the current user or organization.
+   */
+  getApiKeys: (params?: GetAPIKeysParams) => Promise<APIKeyResource[]>;
+
+  /**
+   * This API is in early access and may change in future releases.
+   *
+   * Retrieves the secret for a given API key ID.
+   */
+  getApiKeySecret: (apiKeyID: string) => Promise<string>;
+
+  /**
+   * This API is in early access and may change in future releases.
+   *
+   * Creates a new API key.
+   */
+  createApiKey: (params: CreateAPIKeyParams) => Promise<APIKeyResource>;
+
+  /**
+   * This API is in early access and may change in future releases.
+   *
+   * Revokes a given API key by ID.
+   */
+  revokeApiKey: (params: RevokeAPIKeyParams) => Promise<APIKeyResource>;
```

Comment:
Let's add the `@experimental` JSDoc here, too

Reply:
added!